internal/export/database/export.go

```diff
@@ -683,6 +683,38 @@ func (db *ExportDB) FinalizeBatch(ctx context.Context, eb *model.ExportBatch, fi
 	})
 }
 
+// MarkExpiredFiles marks files for deletion.
+func (db *ExportDB) MarkExpiredFiles(ctx context.Context, configID int64, ttl time.Duration) (int, error) {
+	var filesToDelete int
+	return filesToDelete, db.db.InTx(ctx, pgx.ReadCommitted, func(tx pgx.Tx) error {
+		minTime := time.Now().Add(-1 * ttl)
+		res, err := tx.Exec(ctx, `
+		UPDATE
+			ExportFile AS ef
+		SET
+			status = $5
+		FROM
+			ExportBatch AS eb
+		WHERE
+			eb.config_id = $1
+		AND
+			ef.batch_id = eb.batch_id
+		AND
+			eb.start_timestamp < $2
```

Comment:
this is inconsistent w/ current cleanup, see DeleteFilesBefore

Reply:
current cleanup based on batch end_timestamp, not start

Reply:
my mistake. fixed.